src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -403,7 +410,10 @@ static inline void cavs_pm_runtime_dis_dsp_pg(uint32_t index)
 #if CAVS_VERSION >= CAVS_VERSION_1_8
 	struct pm_runtime_data *prd = pm_runtime_data_get();
 	struct cavs_pm_runtime_data *pprd = prd->platform_data;
-	uint32_t lps_ctl, tries = PLATFORM_PM_RUNTIME_DSP_TRIES;
+	uint32_t lps_ctl;
+#ifndef __ZEPHYR__
+	uint32_t tries = PLATFORM_PM_RUNTIME_DSP_TRIES;
+#endif
```

Comment:
Just move this to the top of `else` block; otherwise this  one-line `#ifdef` is ridiculous. As long as it is at the top of a `{ }` block it does not require C99.
https://wiki.sei.cmu.edu/confluence/display/c/DCL19-C.+Minimize+the+scope+of+variables+and+functions
>  Variables and functions should be declared in the minimum scope from which all references to the identifier are still possible. When a larger scope than necessary is used, code becomes less readable, harder to maintain, and more likely to reference unintended variables

Reply:
Fixed!